bears/css/CSSCombBear.py

```diff
@@ -0,0 +1,76 @@
+                       '.csscomb.json',
+        )
+class CSSCombBear:
+    """
+    CSScomb is a coding style formatter for CSS.
+
+    For more information, consult <https://github.com/csscomb/csscomb.js>.
+    """
+
+    LANGUAGES = {"CSS"}
+    REQUIREMENTS = {NpmRequirement('csscomb')}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    ASCIINEMA_URL = 'https://asciinema.org/a/bxpke7nizyxdf6mlx6ss4h405'
+    CAN_FIX = {'Formatting'}
+
+    @staticmethod
+    def create_arguments(filename, file, config_file):
+        return (filename,)
+
+    def generate_config(filename, file,
+                        verbose: bool = True,
+                        remove_empty_rulesets: bool = True,
+                        always_semicolon: bool = True,
+                        color_case: str = "upper",
+                        block_indent: str = "\\t",
+                        color_shorthand: bool = False,
+                        eof_newline: bool = True,
+                        leading_zero: bool = False,
+                        quotes: str = "single",
+                        sort_order_fallback: str = "abc",
+                        space_before_colon: str = " ",
+                        space_after_colon: str = " ",
+                        space_between_declar: str = "\\n",
+                        space_before_o_brace: str = "\\n",
+                        space_after_o_brace: str = "\\n",
+                        space_after_s_delim: str = "\\n",
+                        space_before_s_delim: str = " ",
+                        space_before_c_brace: str = "\\n",
+                        strip_spaces: bool = True,
+                        tab_size: bool = True,
+                        vendor_prefix_align: bool = True,
+                        csscomb_json: str = ""):
+
+        options = {"remove-empty-rulesets": remove_empty_rulesets,
+                   "always-semicolon": always_semicolon,
+                   "color-case": color_case,
+                   "block-indent": block_indent,
+                   "eof-newline": eof_newline,
+                   "leading-zero": leading_zero,
+                   "quotes": quotes,
+                   "sort-order-fallback": sort_order_fallback,
+                   "space-before-colon": space_before_colon,
+                   "space-after-colon": space_after_colon,
+                   "space-between-declarations": space_between_declar,
+                   "space-before-opening-brace": space_before_o_brace,
+                   "space-after-opening-brace": space_after_o_brace,
+                   "space-after-selector-delimiter": space_after_s_delim,
+                   "space-before-selector-delimiter": space_before_s_delim,
+                   "space-before-closing-brace": space_before_c_brace,
+                   "strip-spaces": strip_spaces,
+                   "vendor-prefix-align": vendor_prefix_align
```

Comment:
You also need to document these parameters, as i have no idea what some do. Take this as an example: https://github.com/coala/coala-bears/blob/master/bears/coffee_script/CoffeeLintBear.py